### PR TITLE
Added changes for sending activity feed notification using rsc permissions

### PR DESCRIPTION
**Pull request by Prithvi-MSFT**

### CODE REVIEW

**Review by mehakagarwal — APPROVED** [review]

File: samples/graph-rsc/nodeJs/Readme.md [hunk 1]
```diff
@@ -99,6 +99,17 @@ The simplest way to run this sample in Teams is to use Teams Toolkit for Visual
...
+
+**Select Reciepient**
+![select-people](./Images/select-people.png)
+
+**Sent Notification**
+![notification](./Images/notification.png)
+
 ## Further Reading.
 
 - [Graph RSC](https://learn.microsoft.com/microsoftteams/platform/graph-api/rsc/resource-specific-consent)
```
Comment (ChetanSharma-msft) [inline samples/graph-rsc/nodeJs/Readme.md:115]: Add link to the concept/feature section

File: samples/graph-rsc/nodeJs/auth.js [hunk 2]
```diff
@@ -13,16 +13,21 @@ auth.getAccessToken = function (tenantId) {
       client_secret: process.env.ClientSecret,  
       scope: 'https://graph.microsoft.com/.default'
     };
+
+    console.log(JSON.stringify(requestParams));
```
Comment (ChetanSharma-msft) [inline samples/graph-rsc/nodeJs/auth.js:17]: Do we need it?

File: samples/graph-rsc/nodeJs/server.js [hunk 3]
```diff
@@ -23,16 +23,20 @@ app.use('/',indexRouter);
...
-      console.log("token from js file : "+ token);
      res.render('./views/rscdemo',{token:JSON.stringify(token)});
     });
-   
+  });
+
+  app.get('/sendNotification', function(req, res) {
+    var tenantId= process.env.TenantId
+    auth.getAccessToken(tenantId).then(async function (token) {
+     res.render('./views/sendNotification',{token:JSON.stringify(token)});
```
Comment (ChetanSharma-msft) [inline samples/graph-rsc/nodeJs/server.js:36]: Check spacing

File: samples/graph-rsc/nodeJs/teamsManifest/manifest.json [hunk 4]
```diff
@@ -15,14 +15,24 @@
...
     "full": "RSC with Graph API"
   },
   "description": {
     "short": "RSC GraphAPI-Node Js",
     "full": "Resource-specific consent (RSC) is a Microsoft Teams and Microsoft Graph API integration that enables your app to use API endpoints to manage specific teams within an organization"
   },
   "accentColor": "#5558AF",
+  "staticTabs": [
+    {
+      "entityId": "?Send Notification",
```
Comment (ChetanSharma-msft) [inline samples/graph-rsc/nodeJs/teamsManifest/manifest.json:28]: Why question mark?

File: samples/graph-rsc/nodeJs/views/sendNotification.ejs [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+
+            microsoftTeams.people.selectPeople({ setSelected: setSelected, openOrgWideSearchInChatOrChannel: openOrgWideSearchInChatOrChannel, singleSelect: singleSelect }).then((people) => {
+                if (people) {
+
+                    let reciepientUserId = people[0].objectId;
+
+                    getInstalledAppList(token, reciepientUserId)
+
+
+                }
```
Comment (ChetanSharma-msft) [inline samples/graph-rsc/nodeJs/views/sendNotification.ejs:41]: Check alignment and spacing.

File: samples/graph-rsc/nodeJs/views/sendNotification.ejs [hunk 6]
```diff
@@ -0,0 +1,125 @@
...
+                },
+                success: function (result) {
+                    appId = getAppId(result.value);
+                    sendActivityFeedNotification(token, reciepientUserId, appId)
+                },
+                error: function (xhr, textStatus, errorThrown) {
+                    console.log("textStatus: " + textStatus + ", errorThrown:" + errorThrown);
+                },
+            });
+
```
Comment (ChetanSharma-msft) [inline samples/graph-rsc/nodeJs/views/sendNotification.ejs:72]: Remove empty spaces from this file and align the contents, if not.

**Review by ChetanSharma-msft — CHANGES_REQUESTED** [review]
Added my review comments.

File: samples/graph-rsc/nodeJs/Readme.md (hunk 1 above)
Comment (Prithvi-MSFT) [reply samples/graph-rsc/nodeJs/Readme.md:117]: added new link

File: samples/graph-rsc/nodeJs/auth.js (hunk 2 above)
Comment (Prithvi-MSFT) [reply samples/graph-rsc/nodeJs/auth.js]: removed

File: samples/graph-rsc/nodeJs/server.js (hunk 3 above)
Comment (Prithvi-MSFT) [reply samples/graph-rsc/nodeJs/server.js]: changed

File: samples/graph-rsc/nodeJs/teamsManifest/manifest.json (hunk 4 above)
Comment (Prithvi-MSFT) [reply samples/graph-rsc/nodeJs/teamsManifest/manifest.json]: removed

File: samples/graph-rsc/nodeJs/views/sendNotification.ejs (hunk 5 above)
Comment (Prithvi-MSFT) [reply samples/graph-rsc/nodeJs/views/sendNotification.ejs:37]: changed

File: samples/graph-rsc/nodeJs/views/sendNotification.ejs (hunk 6 above)
Comment (Prithvi-MSFT) [reply samples/graph-rsc/nodeJs/views/sendNotification.ejs]: changed

**Comment by Prithvi-MSFT** [comment]
Build pipeline - All builds passed
https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/5996855481

**Comment by ChetanSharma-msft** [comment]
image

**Review by Harikrishnan-MSFT — APPROVED** [review]

**Review by ChetanSharma-msft — APPROVED** [review]
Changes looks fine, approving it.

**Pull request merged**